spec/requests/sitemaps_spec.rb

```diff
@@ -28,7 +28,7 @@
     it "sends a surrogate key (for Fastly's user)" do
       articles = create_list(:article, 4)
       included_articles = articles.first(3)
-      included_articles.each { |a| a.update(published_at: "2020-03-07T00:27:30Z", score: 10) }
+      included_articles.each { |a| a.update_columns(published_at: "2020-03-07T00:27:30Z", score: 10) }
```

Comment:
Reading notes: favoring `update_columns` instead of `update` as the `update_columns` avoids validations.